frontends/web/src/decorators/load.tsx

```diff
@@ -40,6 +41,8 @@ export function load<LoadedProps, ProvidedProps = {}>(
         WrappedComponent: ComponentConstructor<LoadedProps & ProvidedProps> | FunctionalComponent<LoadedProps & ProvidedProps>,
     ) {
         return class Load extends Component<ProvidedProps & Partial<LoadedProps>, LoadedProps> {
+            static displayName = `Load(${getDisplayName(WrappedComponent)})`;
+
```

Comment:
I don't get this, where is `static displayName` used?

Reply:
ok nevermind, https://github.com/digitalbitbox/bitbox-wallet-app/pull/56/files#diff-acdabda4a78d1589cb40a111ba587a66R20